backend/import/do_import.py

```diff
@@ -15,6 +15,13 @@
 
 from os2mo_tools import mo_api
 
+SECRET = 'Hemmelig'
+
+
+def is_visible(a):
+    """Decide if address is visible or protected (secret)."""
+    return not ('visibility' in a and a['visibility']['user_key'] == SECRET)
```

Comment:
Perhaps allow a `scope` of `SECRET` as well, so you don't have to change it in the future?